internal/packer/packer.go

```diff
@@ -1432,6 +1436,58 @@ func logic(cfg *config.Struct, programName string) error {
 			fmt.Printf("\n")
 		}
 
+	case cfg.Output.Type == config.OutputTypeGaf && cfg.Output.Path != "":
+		dir, err := os.MkdirTemp("", "gokrazy")
```

Comment:
Can you refactor this into a separate gaf.go please (together with writeGaf)? packer.go is already way too long, so we should probably structure it a little better when we get a chance… :)

Reply:
Yeah had the same feeling :) Moved.